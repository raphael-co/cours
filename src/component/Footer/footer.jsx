import style from './footer.module.css'

function Footer() {

    return (



        <footer className={style.hello}>
            <div className={style.box}>
                <div className={style.one}>
                    <h5>titre 1</h5>
                    <p>lorem ipsum dolor</p>
                    <p>lorem ipsum dolor</p>
                    <p>lorem ipsum dolor</p>
                    </div>
                <div className={style.two}>
                    <h5>Qui sommes nous</h5>
                    <p>lorem ipsum dolor</p>
                    <p>lorem ipsum dolor</p>
                    <p>lorem ipsum dolor</p>
                </div>
                <div className={style.three}>
                    <h5>nous contacter</h5>
                    <p>lorem ipsum dolor</p>
                    <p>lorem ipsum dolor</p>
                    <p>lorem ipsum dolor</p>
                </div>
            </div>
        </footer>

    )
}
export default Footer;