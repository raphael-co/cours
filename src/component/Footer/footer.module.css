.hello {
    overflow: hidden;
    background-color: #791cf8;
    padding: 20px 10px;
    text-align: center;
    
    }
    .box {
      display: flex;
    }

    .one {
      flex: 1;
    }

    .two {
      flex: 1;
    }

    .three {
      flex: 1;
    }