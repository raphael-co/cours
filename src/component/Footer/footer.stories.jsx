import Footer from './footer.jsx'
export default {
    title: 'component/Footer',
    component : 'Footer'
}

export const Normal = () => <Footer></Footer> 