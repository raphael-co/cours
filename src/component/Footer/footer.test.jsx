import { render, screen } from "@testing-library/react"
import Footer from "./footer.jsx"

test("print hello world to the user", () => {
    render(<Footer/>);

    const foot = screen.getByText("titre 1");
    expect(foot).toBeInTheDocument();
    expect(foot).toMatchSnapshot();
})