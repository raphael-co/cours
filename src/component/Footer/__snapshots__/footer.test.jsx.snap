// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`print hello world to the user 1`] = `
<h5>
  titre 1
</h5>
`;
